tests/Phrasing/LabelTest.php

```
@@ -0,0 +1,560 @@
+            <label for="email"></label>
+            HTML,
+            LineEndingNormalizer::normalize(
+                Label::tag()->for('email')->render(),
+            ),
+            "Failed asserting that element renders correctly with 'for' attribute.",
+        );
+    }
+
+    public function testRenderWithGlobalDefaultsAreApplied(): void
+    {
+        SimpleFactory::setDefaults(Label::class, ['class' => 'default-class']);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Factory\SimpleFactory' in method 'testRenderWithGlobalDefaultsAreApplied'. (undefined)

(StaticAccess)